code/control/DNRoot.php

```diff
@@ -88,12 +89,14 @@ class DNRoot extends Controller implements PermissionProvider, TemplateGlobalPro
 		'gitRevisions',
 		'deploySummary',
 		'startDeploy',
+		'createproject',
+		'CreateProjectForm',
```

Comment:
"The UI is good enough for customers" - acceptance criteria. It will initially be used by Ops but will be rolled out to normal users and be part of the trial system.